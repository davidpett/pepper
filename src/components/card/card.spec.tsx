import { render } from '@testing-library/react-native'
import * as React from 'react'
import Card from './card'

describe('Card', () => {
  const props = {
    title: 'hello',
  }

  const { container, getByTestId } = render(<Card {...props} />)

  test('snapshot', () => {
    expect(container).toMatchSnapshot()
  })
  test('render', () => {
    expect(getByTestId('cardContainer')).toBeTruthy()
    expect(getByTestId('cardTitle').props.children).toBe('hello')
  })
})
