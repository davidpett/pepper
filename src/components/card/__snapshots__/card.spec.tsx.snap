// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.7}
    style={
      Object {
        "backgroundColor": "#cccccc",
        "borderRadius": 8,
        "height": 100,
        "marginHorizontal": 8,
        "width": 150,
      }
    }
    testID="cardContainer"
  >
    <Text
      style={Object {}}
      testID="cardTitle"
    >
      hello
    </Text>
  </TouchableOpacity>
</View>
`;
